setup.py

```diff
@@ -1,34 +1,70 @@
 """
-Setup module for trustymail
+setup module for trustymail
 
 Based on:
 
-- https://github.com/dhs-ncats/pshtt
+- https://packaging.python.org/distributing/
+- https://github.com/pypa/sampleproject/blob/master/setup.py
 """
 
 from setuptools import setup
+from trustymail import __version__
```

Comment:
This PR is going to require #17 first, since you're using `__version__` here before it is defined there. (In this branch there is no `__version__`).